docs/topics/collection-grouping.md

```diff
@@ -39,12 +39,37 @@ Namely, `Grouping` supports the following operations:
    This is the generic way to perform any operations on a `Grouping`. Use it to implement custom operations when fold or reduce are not enough.
 
 ```kotlin
-
 fun main() {
 //sampleStart
     val numbers = listOf("one", "two", "three", "four", "five", "six")
     println(numbers.groupingBy { it.first() }.eachCount())
 //sampleEnd
 }
 ```
-{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
\ No newline at end of file
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To iterate through the groups created by the `groupBy()` function, you can use the [`forEach`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each.html) function on the resulting `Map`.
```

Comment:
`for` operator works as well: https://kotlinlang.org/docs/coding-conventions.html#loops

Reply:
I added it 👍